src/Resources/config/core.xml

```diff
@@ -31,7 +31,7 @@
             <argument type="service" id="sonata.block.renderer"/>
             <argument type="service" id="sonata.block.context_manager"/>
             <argument type="service" id="event_dispatcher"/>
-            <argument type="service" id="sonata.cache.manager" on-invalid="ignore"/>
+            <argument type="service" id="TODO" on-invalid="ignore"/>
```

Comment:
I dunno what to use as id for the symfony cache, help @sonata-project/contributors

Reply:
sonata.symfony_cache

Reply:
I don't think that's right, sonata shouldn't be in the id I'm looking for.
Event dispatcher from symfony has id `event_dispatcher`.
Here, I'm looking for the symfony service implementing the `CacheItemPoolInterface` interface.

Reply:
I also need to replace the `sonata.cache.noop` occurences.
https://github.com/sonata-project/SonataBlockBundle/search?q=sonata.cache.noop

Reply:
My bad. In this place we should inject service by alias. Then alias can be configure in files core/legacy_core. Also deprecation can be easy add in legacy_core. In 5.x corrrct service can be inject directly. Then `sonata.cache_manager_alias` can be used

Reply:
I think I made a huge mistake thinking CacheManagerInterface could be replaced by CacheItemPoolInterface.
But a CacheManagerInterface returns CacheServices, and CacheServices can be replaced by CacheItemPoolInterface.

So the cache_manager_alias, should be something like the container in the symfony case. This way with a service id, it returns the cache service with this id. But still I don't know what's the "recorder" in this case and how to deal with extra_cache_keys...